packages/wallet/test/helpers.ts

```diff
@@ -106,14 +110,15 @@ export const calculateSwapWithFees = (
   )
 }
 
-export const assertThrows = (func: () => any, errorType: any): void => {
+export const assertThrows = (func: () => any, errorType?: any): void => {
```

Comment:
Just curious if this has any different functionality from `should.throw()` :)

Reply:
Probably not, but I think I created this when creating the async version because `should.throw()` can't handle that. Should be safe to remove this one, though.